misc/scripts/search.sh

```diff
@@ -75,6 +75,9 @@ fi
 PACKAGELIST=()
 URLLIST=()
 while IFS= read -r URL; do
+	if ! check_url "${URL}/packagelist"; then
+		exit 1
```

Comment:
Add warning for the invalid repo

Reply:
An invalid repo will return the error code 404.